staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/aggregator/priority.go

```diff
@@ -18,56 +18,60 @@ package aggregator
 
 import (
 	"sort"
+
+	v1 "k8s.io/kube-aggregator/pkg/apis/apiregistration/v1"
 )
 
 // byPriority can be used in sort.Sort to sort specs with their priorities.
 type byPriority struct {
-	specs           []openAPISpecInfo
+	apiServices     []v1.APIService
```

Comment:
These objects embed ObjectMeta which is pretty massive. Can we turn that into a pointer to avoid moving large chunks of data around?